DQM/HcalTasks/plugins/HcalGPUComparisonTask.cc

```diff
@@ -106,16 +106,18 @@ HcalGPUComparisonTask::HcalGPUComparisonTask(edm::ParameterSet const& ps)
   _currentLS = lumiCache->currentLS;
 
   std::map<HcalDetId, double> mRecHitEnergy;
+  std::map<HcalDetId, HBHERecHit> mRecHit;
 
   for (HBHERecHitCollection::const_iterator it = chbhe_ref->begin(); it != chbhe_ref->end(); ++it) {
     double energy = it->energy();
 
     //	Explicit check on the DetIds present in the Collection
     HcalDetId did = it->id();
 
-    if (mRecHitEnergy.find(did) == mRecHitEnergy.end())
+    if (mRecHitEnergy.find(did) == mRecHitEnergy.end()) {
       mRecHitEnergy.insert(std::make_pair(did, energy));
-    else
+      mRecHit.insert(std::make_pair(did, *it));
```

Comment:
It seems that `mRecHit` is being filled, but never used ?

Reply:
Removed in 715105876f8999402d3e30f2fa94579275479978